linux_os/guide/system/permissions/files/permissions_var_log_dir/file_owner_var_log_auth/rule.yml

```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: 'Verify User Who Owns /var/log/auth.log File'
+
+description: '{{{ describe_file_owner(file="/var/log/auth.log", owner="syslog") }}}'
```

Comment:
This is inconsistent with the OVAL check for this rule. The description says that the file should be owned by "syslog" user but the OVAL allows both "syslog" and "root" user.

Reply:
Fixed